executor/src/witgen/machines/double_sorted_witness_machine.rs

```diff
@@ -61,22 +61,22 @@ impl DoubleSortedWitnesses {
 }
 
 impl Machine for DoubleSortedWitnesses {
-    fn process_plookup(
+    fn process_plookup<'a>(
         &mut self,
-        fixed_data: &FixedData,
+        _fixed_data: &FixedData,
```

Comment:
Why can't it just be removed?

Reply:
This is a trait function and some other machines still need it I think. I can check.

Reply:
Yes, the BlockMachine still needs it because it needs to evaluate fixed columns on specific rows.

Reply:
fine with leaving, was just curious